src/actions/ticket-actions.js

```diff
@@ -483,7 +483,11 @@ export const getTickets =
       per_page: perPage,
       access_token: accessToken,
       expand:
-        "owner,order,ticket_type,badge,badge.type,promo_code,promo_code.tags,refund_requests,refund_requests.refunded_taxes"
+        "owner,ticket_type,badge,badge.type,promo_code,promo_code.tags,refund_requests",
```

Comment:
re: badge.type i dont think u need to expand it
bc u have the current summit and there all available badge types
u only need the id and find it on the current summit

Reply:
also please re review if all fields of ticket list are really needed
i am seeing at ticket level
    "qr_code": "TICKET_2024OCPGLO|TICKET_2024OCPGLO_66E9D1B17BA42706257624",
and i don think that we use it on grid